### DISCUSSION TITLE
Limiting editors

### DISCUSSION THREAD

**Discussion by schedules000** [code4recovery/12-step-meeting-list#1602] [Q&A]
I'd like to limit editors to the districts they're from so that the rep from District 20 can just edit meetings in District 20.  Can that be done"?

**Comment by SIA-WEBSUP** [comment]
Here is a [thread regarding roles and how to limit permissions ](https://github.com/code4recovery/12-step-meeting-list/discussions/1374#discussioncomment-8670137)that may answer some of your questions.

**Comment by joshreisner** [comment]
yes, you can do this! assign the district 20 user an `author` role - any meeting they create will be editable by them. you (as an admin) can change the author of existing meetings using the author panel. if it's not showing, go under "Screen Options" to show it. you'll want to make sure that all the district 20 meetings have the district 20 user selected.

![Screenshot 2024-12-31 at 19-09-11 Edit Meeting “Back To Basics” ‹ Code for Recovery — WordPress](https://github.com/user-attachments/assets/e0494497-820f-4dd0-a2c5-08cec3366391)

no other plugin should be necessary, but installing other plugins may help more finely-tune user permissions

**Comment by schedules000** [comment]
Thanks Josh.  You're the best! Happy New Year.

On Tue, Dec 31, 2024 at 10:14 PM Josh Reisner ***@***.***>
wrote:

> yes, you can do this! assign the district 20 user an author role - any
> meeting they create will be editable by them. you (as an admin) can change
> the author of existing meetings using the author panel. if it's not
> showing, go under "Screen Options" to show it
>
> Screenshot.2024-12-31.at.19-09-11.Edit.Meeting.Back.To.Basics.Code.for.Recovery.WordPress.png
> (view on web)
> <https://github.com/user-attachments/assets/e0494497-820f-4dd0-a2c5-08cec3366391>
>
> no other plugin should be necessary, but installing other plugins may help
> more finely-tune user permissions
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/1602#discussioncomment-11708673>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BODEYHCQTCY6CLPLCOT56PL2INMS7AVCNFSM6AAAAABUNX624WVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNZQHA3DOMY>
> .
> You are receiving this because you authored the thread.Message ID:
> <code4recovery/12-step-meeting-list/repo-discussions/1602/comments/11708673
> @github.com>
>


-- 
*Area 11 Schedules Chair*
